mkstage4.sh

```diff
@@ -12,23 +12,23 @@ EXCLUDE_BOOT=0
 EXCLUDE_CONNMAN=0
 EXCLUDE_LOST=0
 QUIET=0
-USER_EXCL=""
+USER_EXCL=()
 S_KERNEL=0
 x86_64=0
 PARALLEL=0
 HAS_PORTAGEQ=0
 
-if [ `which portageq` ]
+if command -v portageq &>/dev/null
```

Comment:
What is wrong with `which`?

Reply:
Shellcheck says to avoid which because it is non-standard (BSD and GNU differ in arguments and output) and command is built in to Bash.

https://github.com/koalaman/shellcheck/wiki/SC2230